app-i18n/uchardet/uchardet-9999.ebuild

```diff
@@ -7,10 +7,7 @@ inherit cmake-utils git-r3
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-EGIT_REPO_URI=(
-	https://anongit.freedesktop.org/git/${PN}/${PN}.git
-	git://anongit.freedesktop.org/${PN}/${PN}
-)
+EGIT_REPO_URI="https://anongit.freedesktop.org/git/${PN}/${PN}.git"
```

Comment:
Don't do this. If you personally need security, disable git:// protocol on your machine and be happy. HTTP{,S} aren't handled by plain git and requires git[curl]. HTTPS is blocked in some places, e.g. public cafes, public networks in universities, etc.